backend/src/LeanCode.AppRating/Handlers/SubmitAppRatingCH.cs

```diff
@@ -59,12 +58,13 @@ AppRatingReportsConfiguration appRatingReportsConfiguration
 
     public async Task ExecuteAsync(HttpContext context, SubmitAppRating command)
     {
-        var userId = extractor.Extract(context);
+        var userId = extractor.TryExtract(context, out var uid) ? uid : default;
 
         store
             .AppRatings
             .Add(
                 new AppRating<TUserId>(
+                    new AppRatingId(),
```

Comment:
That would be empty/zero/default, wouldn't it?

```suggestion
                    AppRatingId.New(),
```